app/controllers/application_controller.rb

```diff
@@ -1,5 +1,16 @@
 class ApplicationController < ActionController::Base
-  def current_user
-    User.first
+  protect_from_forgery with: :exception
+
+  before_action :update_allowed_parameters, if: :devise_controller?
+
```

Comment:
- [ ] It looks like we are not done with setting up `devise`. Please ensure that all users get authenticated before taking any action in the App by adding this 👇🏿:

```ruby
before_action :authenticate_user!
```

With this `devise` will now serve its purpose of securing the app. Currently devise is installed but not properly setup to secure the app.